quack_app/lib/screens/homepage/home_screen.dart

```diff
@@ -0,0 +1,125 @@
+import 'dart:ui';
+
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:quack_app/components/navbar.dart';
+import 'package:quack_app/constants/constants.dart';
+import 'package:quack_app/core/menu_data.dart';
+
+class HomeScreen extends StatefulWidget {
+  final String title;
+  const HomeScreen({Key? key, required this.title}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  late MenuData menuData;
+  @override
+  void initState() {
+    // Re-cache updated values on view
+    super.initState();
+    menuData = MenuData();
+  }
+
+  void loadData() async {
```

Comment:
Same for here, is there a way to get tests?

Reply:
Again, nice catch! loadData calls all the async methods, we could test accessing the variables defined as late and try to expect an error. Then loadData and expect the values. Not sure if this will work -- will look into it!

Reply:
Same here - There doesn't seem to be a way to test this without getting messy :(